docs/runner-type/runner-type.md

```diff
@@ -0,0 +1,5 @@
+# Runner Types
+
+Metrics are labelled with the runner type. The logic for determining the runner type is as follows -
+
+If the runner tag is in the [github_hosted_runner_labels](./gh_actions_exporter/config.py) list the runner type will be `github-hosted` else the runner type will be `self-hosted`. Additional `github-hosted` runner tags can be added via config.
```

Comment:
Not sure how the doc was rendered (through mkdocs or github), through github at least couldn't get it to redirect unless I go up a couple of folders.

Also added some note for my future self as I was wondering how we could handle GitHub hosted larger runners with this setting.

```suggestion
- If the runner tag is in the [github_hosted_runner_labels](../../gh_actions_exporter/config.py)
  list, the runner type will be `github-hosted` else the runner type will be `self-hosted`.
- Additional `github-hosted` runner tags can be added via the config file,
  for example GitHub hosted larger runners as their name is defined
  by the owner of the GitHub organization.
```